parser/parser_test.go

```diff
@@ -366,7 +366,7 @@ func TestFromFile(t *testing.T) {
 				// Workflow "name" no longer a required property
 				assert.Empty(t, w.Name)
 				// 	Workflow "start" no longer a required property
-				assert.Empty(t, w.Start)
+				assert.Equal(t, w.States[0].Name, w.Start.StateName)
```

Comment:
see my comment before.

Reply:
That test is changed why when populating the BaseWorkflow.Start with the first state the test fails.

https://github.com/serverlessworkflow/sdk-go/pull/156/files/49add33759f3b194d9f4060a8fd3efdf6900b8b6#diff-6a5d032df78474ceeda4fabb343d1ee198295867c5339285f03e7e65459f5985R215-R221

Reply:
It's better to not do this assertation here since people can get confused.